xml/jvm/src/test/scala/scala/xml/ReuseNodesTest.scala

```diff
@@ -10,19 +10,19 @@ import org.junit.experimental.theories.DataPoints
 import org.junit.runner.RunWith
 /**
  * This test verify that after the tranform, the resultant xml node
- * uses as many old nodes as possible. 
- * 
- * Three transformers class for case - 
- * One for orginal, one for modified, and one proposed which shows 
+ * uses as many old nodes as possible.
+ *
+ * Three transformers class for case -
+ * One for orginal, one for modified, and one proposed which shows
  * all are equivalent when it comes to reusing as many nodes as possible
```

Comment:
If the only changes to this file, ReuseNodesTests.scala, are changes to whitespace, maybe revert so as to avoid touching the file besides the rename.  We can fix whitespace in another PR